src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Button = ({ name, styles }) => {
```

Comment:
As the general use of a button in the project is not for **form submission**. You may change the type from `"Submit"` to `type="button"` unless it is explicitly mentioned.

Reply:
Sure